src/test/java/wooteco/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,221 @@
+package wooteco.subway.line;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import wooteco.subway.AcceptanceTest;
+import wooteco.subway.line.dto.LineResponse;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("노선 관련 기능")
+class LineAcceptanceTest extends AcceptanceTest {
+
+    @DisplayName("노선 생성 - 성공")
+    @Test
+    void createLine() {
+        // given
+        Map<String, String> lineInfo = 노선_정보("신분당선", "bg-red-600");
+
+        // when
+        ExtractableResponse<Response> response = 노선_생성(lineInfo);
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        assertThat(response.header("Location")).isNotBlank();
+        final LineResponse lineResponse = response.body().as(LineResponse.class);
+        assertThat(lineResponse.getName()).isEqualTo("신분당선");
+        assertThat(lineResponse.getColor()).isEqualTo("bg-red-600");
+    }
+
+    @DisplayName("노선 생성 - 실패(이름 중복)")
+    @Test
+    void createLine_duplicatedName() {
+        // given
+        Map<String, String> lineInfo = 노선_정보("신분당선", "bg-red-600");
+        노선_생성(lineInfo);
+
+        // when
+        ExtractableResponse<Response> response = 노선_생성(lineInfo);
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
+        assertThat(response.header("Location")).isBlank();
+    }
+
+    @DisplayName("노선 목록 조회 - 성공")
+    @Test
+    void getLines() {
+        /// given
+        Map<String, String> lineInfo1 = 노선_정보("신분당선", "bg-red-600");
+        final ExtractableResponse<Response> createResponse1 = 노선_생성(lineInfo1);
```

Comment:
테스트 코드도 유지보수해야되는 코드입니다.
읽기 좋은 코드를 작성해보아요

```suggestion
        final ExtractableResponse<Response> 신분당선 = 노선_생성(lineInfo1);
```